text/0000-new-maintainer-group-process.md

```diff
@@ -0,0 +1,22 @@
+# Codify the Creation of New Maintainer Groups
+
+## Summary
+
+This RFC is intended to define processes for the creation of new maintainer groups.
+
+## Motivation
+
+Currently the creation of new maintainer groups is an ad hoc process that has to be hashed out every time a new group is requested. This means that intervention from the Steering Committee is often needed in situations where they should not need to weigh in. It also means that there is no standard record for the introduction of new groups meaning that the reasoning behind the formation of these new maintainer groups is lost. Codifying this process should make it easier to identify when new maintainer groups should be formed and make the process of doing so much easier. It should also give approval power to the stakeholders that will be directly affected by the introduction of new maintainer groups, namely the existing maintainer groups as opposed to the Steering Committee.
```

Comment:
Has there been a case where this is an issue? I'm not really sure I see the problem that this addresses. We create groups pretty infrequently, and from what I can recall, that's been done via RFC. Is there a case where an official group was formed without an RFC?

Reply:
I think that most of our maintainer groups were created before a formalized RFC process existed. I have had a couple of people approach me asking if there was an official process. I figured that formalizing the process made sense and would clear up question of voting going forward.

Reply:
OK. Sounds reasonable. I like the idea of having transparent processes.

I would suggest that we call out where this process is going to live. Probably add something to the Implementation section that says "Add a page to the website to document this process". People shouldn't have to dig through RFCs to understand the process.